### ISSUE TITLE
[BUG]: Issue with new Lago Docker Compose Installation

### ISSUE THREAD

**Issue by abdofallah** [getlago/lago#373]
Unable to make Lago work with a fresh/new Docker Compose installation.

Able to load the login/sign up page but sign up fails with the following error:
![image](https://github.com/getlago/lago/assets/38984510/d585723a-e2f2-4bdf-b083-4ee884679834)
upon visiting api/graphql, this is what i see:
![image](https://github.com/getlago/lago/assets/38984510/1401994e-5563-4236-aaf3-f82e1d4c4a95)

The lago-worker seems to be in a constant restart loop. Logs:
```
super: no superclass method `broadcast' for class ActiveSupport::Logger
/usr/local/bundle/gems/newrelic_rpm-9.5.0/lib/new_relic/agent/instrumentation/active_support_logger/prepend.rb:9:in `block in broadcast'
/usr/local/bundle/gems/newrelic_rpm-9.5.0/lib/new_relic/agent/instrumentation/active_support_logger/instrumentation.rb:16:in `broadcast_with_tracing'
/usr/local/bundle/gems/newrelic_rpm-9.5.0/lib/new_relic/agent/instrumentation/active_support_logger/prepend.rb:9:in `broadcast'
/usr/local/bundle/gems/sidekiq-7.1.4/lib/sidekiq/rails.rb:52:in `block (2 levels) in <class:Rails>'
/usr/local/bundle/gems/sidekiq-7.1.4/lib/sidekiq.rb:98:in `configure_server'
/usr/local/bundle/gems/sidekiq-7.1.4/lib/sidekiq/rails.rb:47:in `block in <class:Rails>'
/usr/local/bundle/gems/railties-7.1.3.4/lib/rails/initializable.rb:32:in `instance_exec'
/usr/local/bundle/gems/railties-7.1.3.4/lib/rails/initializable.rb:32:in `run'
/usr/local/bundle/gems/railties-7.1.3.4/lib/rails/initializable.rb:61:in `block in run_initializers'
/usr/local/lib/ruby/3.3.0/tsort.rb:231:in `block in tsort_each'
/usr/local/lib/ruby/3.3.0/tsort.rb:353:in `block (2 levels) in each_strongly_connected_component'
/usr/local/lib/ruby/3.3.0/tsort.rb:434:in `each_strongly_connected_component_from'
/usr/local/lib/ruby/3.3.0/tsort.rb:352:in `block in each_strongly_connected_component'
/usr/local/lib/ruby/3.3.0/tsort.rb:350:in `each'
/usr/local/lib/ruby/3.3.0/tsort.rb:350:in `call'
/usr/local/lib/ruby/3.3.0/tsort.rb:350:in `each_strongly_connected_component'
/usr/local/lib/ruby/3.3.0/tsort.rb:229:in `tsort_each'
/usr/local/lib/ruby/3.3.0/tsort.rb:208:in `tsort_each'
/usr/local/bundle/gems/railties-7.1.3.4/lib/rails/initializable.rb:60:in `run_initializers'
/usr/local/bundle/gems/railties-7.1.3.4/lib/rails/application.rb:426:in `initialize!'
/app/config/environment.rb:7:in `<top (required)>'
/usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `require'
/usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `block (2 levels) in replace_require'
/usr/local/bundle/gems/sidekiq-7.1.4/lib/sidekiq/cli.rb:303:in `boot_application'
/usr/local/bundle/gems/sidekiq-7.1.4/lib/sidekiq/cli.rb:42:in `run'
/usr/local/bundle/gems/sidekiq-7.1.4/bin/sidekiq:31:in `<top (required)>'
/usr/local/bundle/bin/sidekiq:25:in `load'
/usr/local/bundle/bin/sidekiq:25:in `<top (required)>'
/usr/local/bundle/gems/bundler-2.5.5/lib/bundler/cli/exec.rb:58:in `load'
/usr/local/bundle/gems/bundler-2.5.5/lib/bundler/cli/exec.rb:58:in `kernel_load'
/usr/local/bundle/gems/bundler-2.5.5/lib/bundler/cli/exec.rb:23:in `run'
/usr/local/bundle/gems/bundler-2.5.5/lib/bundler/cli.rb:451:in `exec'
/usr/local/bundle/gems/bundler-2.5.5/lib/bundler/vendor/thor/lib/thor/command.rb:28:in `run'
/usr/local/bundle/gems/bundler-2.5.5/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/usr/local/bundle/gems/bundler-2.5.5/lib/bundler/vendor/thor/lib/thor.rb:527:in `dispatch'
/usr/local/bundle/gems/bundler-2.5.5/lib/bundler/cli.rb:34:in `dispatch'
/usr/local/bundle/gems/bundler-2.5.5/lib/bundler/vendor/thor/lib/thor/base.rb:584:in `start'
/usr/local/bundle/gems/bundler-2.5.5/lib/bundler/cli.rb:28:in `start'
/usr/local/bundle/gems/bundler-2.5.5/exe/bundle:28:in `block in <top (required)>'
/usr/local/bundle/gems/bundler-2.5.5/lib/bundler/friendly_errors.rb:117:in `with_friendly_errors'
/usr/local/bundle/gems/bundler-2.5.5/exe/bundle:20:in `<top (required)>'
/usr/local/bundle/bin/bundle:25:in `load'
/usr/local/bundle/bin/bundle:25:in `<main>'
```

lago-api logs:
```
Unrecognized command "db:migrate:primary" (Rails::Command::UnrecognizedCommandError)
Did you mean?  db:migrate:up
=> Booting Puma
=> Rails 7.1.3.4 application starting in production 
=> Run `bin/rails server --help` for more startup options
RecurringTransactionRule's `field :method` conflicts with a built-in method, use `resolver_method:` to pick a different resolver method for this field (for example, `resolver_method: :resolve_method` and `def resolve_method`). Or use `method_conflict_warning: false` to suppress this warning.
Puma starting in single mode...
* Puma version: 6.4.2 (ruby 3.3.0-p0) ("The Eagle of Durango")
*  Min threads: 0
*  Max threads: 5
*  Environment: production
*          PID: 16
* Listening on http://[::]:3000
Use Ctrl-C to stop
```

It seems the migration is failing at lago-api too.

**Comment by abdofallah** [comment]
the signup was fixed by running these commands manually:

```
rails db:create
rails db:migrate
```

however, the api-worker still keeps restarting with the above logs.

**Comment by jdenquin** [comment]
@abdullahbaa5 can you try to start a fresh container for worker?

**Comment by willymwai** [comment]
@jdenquin The issue affects sidekiq 7.1.4. Bumping it to 7.1.5 fixes the issue.

https://github.com/sidekiq/sidekiq/issues/6063

I've created a PR to fix the issue here -> https://github.com/getlago/lago-api/pull/2321

**Comment by jdenquin** [comment]
This is fixed on the last version `1.8.2`
